### DISCUSSION TITLE
Is it a good idea to use Next.js with react-native-web?

### DISCUSSION THREAD

**Discussion by vasco3** [vercel/next.js#11413] [Help]
I looked at the [nextjs react-native-web example](https://github.com/zeit/next.js/tree/canary/examples/with-react-native-web). I wonder which are the trade-off of implementing this?

I assume that the app would lose the nextjs routing with `next/router` and `<Link>`.
Also I assume that all the style-jsx, css and sass features would be lost.
How would code-splitting and all the other nextjs goodies would be affected?

**Comment by MoOx** [comment]
You can mix both. Main benefit is to reuse UI element between RN app & RNW app made with Next. You can share everything that is not routing.
